doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,151 @@ HAFS Workflow Schematic
+
+HAFSv2 will replace NCEP's current operational hurricane forecast systems, HAFSv1 in the 2024 hurricane season. The goal of this project is to upgrade the HAFSv1 that was running in operations in 2023. The system will provide improved and comparable tropical cyclone track and intensity forecast guidance in all global oceanic basins, as demonstrated through evaluation of retrospective and real time experiments compared against the two current configurations of operational HFSAv1 and HFSBv1.
+
+Scientific Changes to Improve Track and Intensity Forecast Skills
+-----------------------------------------------------------------
+
+**System and Infrastructure Upgrades:**
+    - Latest version of ufs-weather-model, HAFSv2 final scientific configuration freeze on 02/08/2024.
+    - Increase moving nesting horizontal resolution from 6-2 km to 5.4-1.8 km (HFSA only).
+    - Reduce model time step from 90 to 72s (HFSB only).
+    - Updated horizontal advection options.
+    - Improved model stability and runtime efficiency.
+
+**Vortex Initialization Improvement:**
+    - Enhance vortex initialization to cycle hydrometeor variables and vertical velocity (HFSA only).
+    - Update compsitie vortex and reduce warm-cycling Vmax threshold from 50 to 40 kt (HFSA only).
+
+**Data Assimilation Improvement:**
+    - Ingest new high-resolution GOES-R mesoscale AMVs.
+    - Scale-Dependent Localization for innercore DA.
+    - Refine GPS RO (Radio Occultation) DA.
+
+**Model Physics Advancement:**
+    - Upgrade Thompson MP with bug fixes.
+    - Thompson Microphysics for NATL basin, GFDL Microphysics for EPAC/CPAC and JTWC basins (HFSA only).
+    - Update TKE EDMF PBL and SASAS CP schemes with vertical wind shear impacts.
+    - Change the radiation calling time step from 720s to 900s (HFSA only).
+    - Reduce radiation time step from 1800 to 720s (HFSB only).
+    - Update CO2 fix files.
+
+**Ocean/Wave Coupling:**
+    - MOM6 ocean model coupling (HFSA only).
+    - CMEPS with inline-CDEPS coupling (HFSA only).
+
+**Post-processing:**
+    - Upgrade GFDL Tracker.
+
+**Software Dependencies:**
+    - See `here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`__
+
+**Output Changes:**
+    - See `here <https://docs.google.com/presentation/d/1dGWu_k-CdiX_ndaRe89iDznyNwkEJfjVTagvvEVZQaw/edit?usp=sharing>`__
```

Comment:
Lets not link to Google docs (for reasons mentioned above). Any info in a Google doc should be moved to the documentation itself.